client/components/Admin/User/UserPage.tsx

```diff
@@ -180,11 +220,25 @@ const UserPage: FC<{}> = () => {
         setQuery={setQuery}
         search={setSearch}
         move={move}
+        setName={setName}
+        setEmailToBeChanged={setEmailToBeChanged}
+        openUserEditModal={openUserEditModal}
```

Comment:
```suggestion
        openUserEditModal={({user}) => {
          setName(user.name)
          setEmailToBeChanged(user.email)
          openUserEditModal(user)
        }}
```

ってかんじでいけたりしないだろうか (動作未検証)
(そうしたら setName や setEmailToBeChanged を UserTable に渡す必要ないかな〜って

Reply:
できそうなのでやってみます

Reply:
やってみました